src/components/Forms/LocationForm/LocationForm.js

```diff
@@ -135,6 +136,11 @@ const LocationForm = props => {
   );
 };
 
+LocationForm.propTypes = {
+  className: PropTypes.string,
+  setOptions: PropTypes.func
```

Comment:
propTypes are wrong. To check what kind of proptypes it is receiving do CTRL + SHIFT + F search for the component name and see what props it is receiving